openid-federation-1_0.xml

```diff
@@ -530,11 +530,13 @@
               <vspace/>
               OPTIONAL. An array of strings representing
               the Entity Identifiers of Intermediate Entities or Trust Anchors
-              that MAY issue Subordinate Statements about the Entity.
+              that are Immediate Superiors of the Entity.
               This claim is REQUIRED in Entity Configurations of
               the Entities that have at least one Superior above them,
```

Comment:
+ "and Entity Statements"

If this is lacking from Entity Statements, path construction will terminate and treat this entity as the Trust Anchor.